app/src/main/java/com/android/template/utils/auth/AuthUtils.kt

```diff
@@ -0,0 +1,13 @@
+package com.android.template.utils.auth
+
+import android.content.Intent
+
+interface AuthUtils {
```

Comment:
Kada neka klasa ima ime `utils` u sebi, onda je to obično znak da ili nije bilo kreativnosti u kreiranju imena ili da klasa radi par stvari, pa kao "utils" je dovoljno generalno.